python/tvm/relay/frontend/nnvm_common.py

```diff
@@ -108,6 +108,17 @@ def _transpose(inputs, attrs):
     return _op.transpose(inputs[0], axes=axes)
 
 
+def _swap_axis(inputs, attrs):
```

Comment:
I think it's better to define this in mxnet.py as _mx_swapaxis().

Reply:
done.